### ISSUE TITLE
Ooh, Shiny! not giving defense bonus

### ISSUE THREAD

**Issue by Slandor** [eternaldensity/Sandcastle-Builder#1424]
in dragon.js line 471, ooh, shiny! didn't get renamed

if (Molpy.Got('Ooo Shiny!')) Molpy.DragonDefenceMultiplier *= (1+Math.log(Molpy.Level('Gold')));
should be
if (Molpy.Got('Ooh, Shiny!')) Molpy.DragonDefenceMultiplier *= (1+Math.log(Molpy.Level('Gold')));

**Comment by frisco82** [comment]
This is already fixed in trunk but it not released, this is a real stop in the dragon progression. Reaching Wyverns is really difficult without this fix and fighting lords is almost impossible and this also makes harder to progress on masterpieces.

It is documented in the guide that it doesn't work but you have to digg in reddit to get the temporary workaround, so please release this soon as dragon and masterpieces are almost the only thing to do in the EoT before 4000.

**Comment by pickten** [comment]
Agreed. The changes already made for new content aren't detectable (since they're isolated from the rest), so I think it would be good to release these soon and then add the new content next patch.

**Comment by Pikrass** [comment]
ED, would you at least cherry-pick the commit to gh-pages? I'm playing on a local copy of the code with the patch applied because of this issue.

**Comment by eternaldensity** [comment]
Is there anything half done or experimental in master at the moment?  I
should be able to do a merge within the day.

**Comment by pickten** [comment]
Besides bugfixes, the only changes pushed to master are:

 - Support for object/string saving (unused, but works)
 - A new library of Sokoban stuff (unused, but works)
 - A few QoL changes.

So nothing half-done or experimental besides the prep work for the moon tower, but it's (a) working and (b) inaccessible besides console stuff.
